.ci/aarch64/install_rom_aarch64.sh

```diff
@@ -65,31 +70,32 @@ build_uefi()
 	make -C edk2/BaseTools
 
 	build -a AARCH64 -t GCC5 -p edk2/ArmVirtPkg/ArmVirtQemu.dsc -b RELEASE
+
+	[ ! -d ${EFI_DEFAULT_DIR} ] && mkdir -p ${EFI_DEFAULT_DIR}
+	cp ${QEMU_EFI_BUILD_PATH} ${EFI_DEFAULT_DIR} || clean_up_and_die "fail to build uefi for arm64"
+
+	export -n WORKSPACE
 	echo "Info: build uefi successfully"
 
 	popd
 }
 
 prepare_default_uefi()
 {
-	if [ "$ID" == "ubuntu" ] || [ "$ID" == "debian" ]; then
-		sudo apt remove -y qemu-efi-aarch64
-		sudo apt install -y qemu-efi-aarch64
-	else
-		local efi_url="https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd"
-		sudo mkdir -p "${EFI_DEFAULT_DIR}"
-		pushd "${WORKSPACE}" || clean_up_and_die "fail to prepare default uefi."
-		curl -LO ${efi_url}
-		sudo install -o root -g root -m 644 QEMU_EFI.fd ${EFI_DEFAULT_DIR}
-		popd
-	fi
+	echo "will download qemu_efi.fd"
+	local efi_url="https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/QEMU_EFI.fd"
```

Comment:
It would be better to avoid hard-coding the file to download by using a variable for `QEMU_EFI.fd`.

Reply:
OK, use a variable instead.